backend/src/test/java/de/kathahuth/gymast/controller/FamilyControllerTest.java

```diff
@@ -0,0 +1,74 @@
+package de.kathahuth.gymast.controller;
+
+import de.kathahuth.gymast.db.FamilyMongoDb;
+import de.kathahuth.gymast.model.Family;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.*;
+
+import java.util.Arrays;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class FamilyControllerTest {
+
+    @LocalServerPort
+    private int port;
+
+    private String getUrl() {
+        return "http://localhost:" + port + "api/difficulty/groupId";
+    }
+
+    @Autowired
+    private TestRestTemplate testRestTemplate;
+
+    @Autowired
+    private FamilyMongoDb familyMongoDb;
+
+    @BeforeEach
+    public void setup() {
+        familyMongoDb.deleteAll();}
+
+    private Family createPushUpFamily() {
+        return Family.builder()
+                .id("push-up-family")
+                .name("Push Up Family")
+                .childrenIds(Arrays.asList("push-up-category"))
+                .build();
+    }
+
+    @Test
+    @DisplayName("Get Family By Id should return FamilyId")
+    public void getFamilyById(){
+        //Given
+        String familyId = "push-up-family";
+        familyMongoDb.save(createPushUpFamily());
```

Comment:
you may want to consider changing your helper method to take in your ID so you can ensure that you always create an element in line with your given ID